### PR TITLE
docs: review and update components overview guide

### PR DESCRIPTION
**Pull request by yallen011**
* added links to navigate to the create workspace and application and install Angular CLI how-to steps for easy navigation
* added reviewed tag

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by atscott — APPROVED** [review]
LGTM but not sure what @aikidave has in mind for the reviewed date so I've added him to the review here.

**Comment by yallen011** [comment]
Thank you, it wouldn't let me add him myself

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Thanks for this review and update. Would you mind tweaking the commit message?

`docs-infra` is used for the underlying angular.io application and the tooling that generates the docs.

In this case I think it would be enough not to provide a target:

```
docs: review components overview guide
```

File: aio/content/guide/component-overview.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ To view or download the example code used in this topic, see the <live-example><
 
 To create a component, verify that you have met the following prerequisites:
 
-1. Install the Angular CLI.
-1. Create an Angular project.
+1. [Install the Angular CLI.](https://angular.io/guide/setup-local#install-the-angular-cli)
```
Comment (gkalpak) [inline aio/content/guide/component-overview.md:22]:
```suggestion
1. [Install the Angular CLI.](guide/setup-local#install-the-angular-cli)
```

File: aio/content/guide/component-overview.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ To view or download the example code used in this topic, see the <live-example><
...
+1. [Create an Angular workspace](https://angular.io/guide/setup-local#create-a-workspace-and-initial-application) with initial application.
```
Comment (gkalpak) [inline aio/content/guide/component-overview.md:23]:
```suggestion
1. [Create an Angular workspace](guide/setup-local#create-a-workspace-and-initial-application) with initial application.
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Drive-by review 🚗

**Review by aikithoughts — APPROVED** [review]
Thanks for this! LGTM!

Reviewed-for: global-docs-approvers

**Review by petebacondarwin — APPROVED** [review]
Looking good, we just need to sort out the commit messages. I guess you are making these changes via the Github UI? If not and you are working in a local git repository, then could you rebase and squash all the commits, into one that uses the proposed commit title?

If not then one of us could probably fix it up for you.

**Review by gkalpak — APPROVED** [review]

**Comment by yallen011** [comment]
fixed squashed commits and fixed commit message

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**